package.json

```diff
@@ -10,6 +10,7 @@
     "react-dom": "^15.5.3",
     "react-redux": "^5.0.5",
     "react-router-dom": "^4.1.1",
+    "react-select": "^1.0.0-rc.10",
```

Comment:
Safe to use a RC for this?

Reply:
Yes. There is a rewrite going on in `react-select` now.
With the API fairly stable, this should be safe.